proto/one/enterprise/data/post_filtering.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+import "post_filtering_column_definition.proto";
+import "google/protobuf/wrappers.proto";
+
+option csharp_namespace ="ONE.Models.CSharp.Enterprise.Data";
+
+message PostFilteringTable {
+
+    // Table name.
+    string name = 1;
+
+    // Column definitions.
+    repeated PostFilteringColumnDefinition columnDefs = 2;
+
+    // ag-Grid filters JSON
+    google.protobuf.StringValue filters = 3;
+}
+
+message PostFiltering {
```

Comment:
You've got multiple messages in a couple files here, should those be split out into separate files?

Reply:
I put them in the same file for the same reason that we have our singular message and the wrapped repeated message in the same class. Seems to me to make it more clear that there are dependencies between the messages.